index.html

```diff
@@ -12,12 +12,9 @@
     <link rel="stylesheet" href="styles.css" />
   </head>
   <body>
-    <!--Navigation-->
     <nav class="container">
```

Comment:
For higher SEO and semantic reasons, I think it would be better to wrap your top navigation inside the header tag. Semantic HTML makes webpages accessible for mobile devices and for people with disabilities as well. This is because screen readers and browsers are able to interpret the code better.

Reply:
> For higher SEO and semantic reasons, I think it would be better to wrap your top navigation inside the header tag. Semantic HTML makes webpages accessible for mobile devices and for people with disabilities as well. This is because screen readers and browsers are able to interpret the code better.

- [ ] You have done incredibly well. You have used professional, polite, and friendly communication skills 👏 🤝 . The student may not be able to have a clear understanding of `Semantic HTML`. In this case, it is always advised to attach the reference link so that the student can read through it to have a clear understanding ☺️ .